Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -185,7 +185,10 @@ var TouchableNativeFeedback = React.createClass({
 
   _handleResponderMove: function(e) {
     this.touchableHandleResponderMove(e);
-    this._dispatchHotspotUpdate(e.nativeEvent.pageX, e.nativeEvent.pageY);
+    var [elementWidth, elementHeight] = (this.pressInLocation)
```

Comment:
I added the fallback just because I wasn't sure whether _dispatchHotspotUpdate() was important for non-ripple-touchables. I would rather stop calling the hotspot update if pressInLocation is undefined than renaming the variables?